com.unity.multiplayer.mlapi/Tests/Runtime/com.unity.multiplayer.mlapi.runtimetests.asmdef

```diff
@@ -1,12 +1,24 @@
 {
     "name": "Unity.Multiplayer.MLAPI.RuntimeTests",
+    "rootNamespace": "",
     "references": [
         "Unity.Multiplayer.MLAPI.Runtime",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner",
         "Unity.Multiplayer.MLAPI.Editor"
     ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
-    ],
     "includePlatforms": [],
-    "excludePlatforms": []
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "nunit.framework.dll"
+    ],
+    "autoReferenced": false,
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS",
+        "UNITY_EDITOR"
+    ],
+    "versionDefines": [],
+    "noEngineReferences": false
```

Comment:
this diff could probably be much smaller with just this diff:
```diff
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS",
+        "UNITY_EDITOR"
+    ],
```

Reply:
There's a bit more than this, there's also the new references.
Thing is, these get readded every time I make a change to that file. I figured I might as well commit everything, this way future changes will be easier to see.